aur.py

```diff
@@ -61,15 +78,15 @@ def main():
     module = AnsibleModule(
         argument_spec={
             'name': {
-                'required': True,
+                'required': False,
             },
             'state': {
                 'default': 'present',
                 'choices': ['present', 'absent'],
             },
             'tool': {
                 'default': 'pacaur',
-                'choices': ['pacaur', 'yaourt'],
+                'choices': ['pacaur', 'yaourt', 'yay'],
```

Comment:
This should go in a separate PR.

Reply:
You are right I will revert these changes